verification/verify.go

```diff
@@ -103,7 +112,29 @@ func (sv *SegmentVerifier) Verify(params *Params) (*Params, error) {
 		return nil, nil
 	}
 
-	// TODO sig checking; extract from broadcast.go
+	if params.Results != nil {
+		segHashes := make([][]byte, len(params.Results.Segments))
+		for i, segment := range params.Results.Segments {
+
+			data, err := drivers.GetSegmentData(segment.Url)
```

Comment:
We'll already have the data in `params.Renditions` if B is using local memory storage so we could skip the download step in that case.

Reply:
thanks - done in 4def93a69fdbc03ac0623ab0ece10451070649a9